draft-ietf-ccamp-eth-client-te-topo-yang.txt

```diff
@@ -41,115 +43,259 @@ Status of This Memo
+Internet-Draft      ETH Topology Transport YANG Model     September 2023
+
+
+   This document describes a YANG data model for Ethernet networks when
+   used either a client-layer network of an underlay transport network
+   (e.g., an Optical Transport Network (OTN)) or a transport network
+   technology.
+
+   The YANG model defined in this document augments from the TE topology
+   YANG model defined in [RFC8795], and imports from the generic
+   Ethernet types defined in [I-D.ietf-ccamp-client-signal-yang].
+
+   The YANG data model in this document conforms to the Network
+   Management Datastore Architecture defined in [RFC8342].
+
+1.1.  Terminology and Notations
+
+   The following terms are defined in [RFC7950] and are not redefined
+   here:
+
+   *  client
+
+   *  server
+
+   *  augment
+
+   *  data model
 
+   *  data node
 
-2.  Terminology and Notations
+   The following terms are defined in [RFC6241] and are not redefined
+   here:
+
+   *  configuration data
+
+   *  state data
+
+   The terminology for describing YANG data models is found in
+   [RFC7950].
+
+1.2.  Tree Diagram
 
    A simplified graphical representation of the data model is used in
-   this document.  The meaning of the symbols in the YANG data tree
-   presented later in this document is defined in [RFC8340].  They are
-   provided below for reference.
+   Section 3 of this document.  The meaning of the symbols in these
+   diagrams is defined in [RFC8340].
+
+1.3.  Prefix in Data Node Names
+
+   In this document, the names of data nodes and other data model
+   objects are prefixed using the standard prefix associated with the
+   corresponding YANG imported modules, as shown in Table 1.
+
+
+
+Yu, et al.                Expires 15 March 2024                 [Page 3]
+
+Internet-Draft      ETH Topology Transport YANG Model     September 2023
+
+
+   In this document, names of data nodes and other data model objects
+   are prefixed using the standard prefix associated with the
+   corresponding YANG imported modules, as shown in Table 1.
+
+            +============+=======================+===========+
+            | Prefix     | YANG module           | Reference |
+            +============+=======================+===========+
+            | yang       | ietf-yang-types       | [RFC6991] |
+            +------------+-----------------------+-----------+
+            | etht-types | ietf-eth-tran-types   | [RFCYYYY] |
+            +------------+-----------------------+-----------+
+            | nw         | ietf-network          | [RFC8345] |
+            +------------+-----------------------+-----------+
+            | nt         | ietf-network-topology | [RFC8345] |
+            +------------+-----------------------+-----------+
+            | tet        | ietf-te-topology      | [RFC8795] |
+            +------------+-----------------------+-----------+
+            | eth-tet    | ietf-eth-te-topology  | RFCXXXX   |
```

Comment:
same question as on the other pull request for YANG models 
"eth-tet" or "etht" as the model prefix?

Reply:
To align with the resolution of the discussion in PR #6

Reply:
not clear to me the comment, Italo. I agree with Aihua it should be etht in alignement with others , recently the Optical Impairment with "oit"

Reply:
Aligned with the resolution in PR #6 

See: https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-eth-client-te-topo-yang/pull/10/commits/e0703159284b753b704cfb36bac7988e74bc7865